metabase/metabase
# fix: Ensure interactive embedding CTA matches instance features

@metabase-bot backport release-x.49.x

### Description

Ensures that the interactive embedding CTA displays the correct banner for OSS, starter, and pro plans.

### How to verify

On OSS and Starter plans, users should see:
<img width="1115" alt="image" src="https://github.com/metabase/metabase/assets/25306947/fe683ec3-0974-43af-8ad3-c89fe7701f19">


On pro plans, users should see:
<img width="1110" alt="image" src="https://github.com/metabase/metabase/assets/25306947/c7d141ad-ad19-41e6-a1a4-a73d63aa9fe6">

### Checklist
